odoo/models.py

```diff
@@ -6421,9 +6421,9 @@ def __len__(self):
 AbstractModel = BaseModel
 
 class Model(AbstractModel):
-    """ Main super-class for regular database-persisted Odoo models.
+    """ Main super-class for regular database-persisted models.
 
-    Odoo models are created by inheriting from this class::
+    Models are created by inheriting from this class::
```

Comment:
Do we really need to scrub core docstrings? Or just client facing strings?

Reply:
So James found an instance where this is displayed in a notes section of a record. I went to double check and found that it is only visible in debug mode, so I don't think a user will ever be able to see it. I will undo these changes.